src/main/java/fi/metatavu/metaform/server/rest/RealmsApiImpl.java

```diff
@@ -542,6 +631,205 @@ public Response updateEmailNotification(String realmId, UUID metaformId, UUID em
     return createOk(emailNotificationTranslator.translateEmailNotification(emailNotification));
   }
 
+
+  @Override
+  public Response createExportTheme(String realmId, ExportTheme payload) throws Exception {
+    if (!isRealmMetaformAdmin()) {
+      return createForbidden(YOU_ARE_NOT_ALLOWED_TO_UPDATE_METAFORMS);
+    }
+    
+    UUID parentId = payload.getParentId();
+    fi.metatavu.metaform.server.persistence.model.ExportTheme parent = null;
+    UUID loggedUserId = getLoggerUserId();
+    
+    if (parentId != null) {
+      parent = exportThemeController.findExportTheme(parentId);
+      if (parent == null) {
+        return createBadRequest(String.format("Theme %s does not exist", parentId));
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal "Theme %s does not exist" 3 times. ([squid:S1192](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1192))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.metaform-api%3Ametaform-api&pullRequest=49&issues=AWWBxMGxXKENQn1S5jLT&open=AWWBxMGxXKENQn1S5jLT)

[ISSUE_KEY]: AWWBxMGxXKENQn1S5jLT